### PR TITLE
version-1

### PR DESCRIPTION
**Pull request by Serhii-Shtokalo**
- [DEMO LINK](https://Serhii-Shtokalo.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://Serhii-Shtokalo.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,59 @@
...
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="Product photo"
+        class="card__image"
+      >
+
+      <h3 class="card__title">
+				APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
```
[the remark this answers is not in the document]
Comment (Serhii-Shtokalo) [reply src/index.html:27]: Знімок екрана 2022-11-12 о 12 13 41

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Push all files. If you have any questions feel free to ask in Slack

File: src/card.css [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  font-size: 12px;
+  line-height: 18px;
+
+  color: var(--primary-color);
+}
+
+.card__product-code {
+  margin-top: 4px;
+
+  font-weight: 400;
```
Comment (ivanchuk-yurii) [inline src/card.css:33]: it's default value, you don't need it. check in other places

File: src/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (ivanchuk-yurii) [reply src/index.html:27]: change LF to CRLF

File: src/reset.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin: 0;
+  padding: 0;
```
Comment (ivanchuk-yurii) [inline src/reset.css:21]: `h` tags don't have default padding, combine this selector with `p`

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

File: src/reset.css (hunk 3 above)
Comment (Serhii-Shtokalo) [reply src/reset.css:21]: Знімок екрана 2022-11-14 о 17 21 05

Comment (Serhii-Shtokalo) [reply src/reset.css:21]: Коли я міняю LF на СRLF видає помилку

**Pull request closed without merging**